pkg/storkctl/groupsnapshot_test.go

```diff
@@ -0,0 +1,213 @@
+// +build unittest
+
+package storkctl
+
+import (
+	"fmt"
+	"strings"
+	"testing"
+
+	storkv1 "github.com/libopenstorage/stork/pkg/apis/stork/v1alpha1"
+	"github.com/portworx/sched-ops/k8s"
+	"github.com/stretchr/testify/require"
+)
+
+func TestOneGroupSnapshot(t *testing.T) {
+	defer resetTest()
+	name := "test-group-snap"
+	namespace := "test"
+	selectors := map[string]string{"app": "mysql"}
+	preRuleName := "mysql-pre-snap"
+	postRuleName := "mysql-post-snap"
+	restoreNamespaces := []string{namespace, "prod"}
+
+	createGroupSnapshotAndVerify(t, name, namespace,
+		selectors, preRuleName, postRuleName, restoreNamespaces, nil, 99)
+
+	expected := fmt.Sprintf("NAME              STATUS    STAGE     SNAPSHOTS   CREATED\n%s                       0           \n", name)
```

Comment:
Can you split the expected output to multiple lines, one per object? Take a look at the other tests